src/widgets/icons.js

```diff
@@ -61,6 +57,10 @@ export const CloseIcon = ({ size, color }) => <IonIcon name="md-close" size={siz
 CloseIcon.defaultProps = { color: WHITE, size: 36 };
 CloseIcon.propTypes = { color: PropTypes.string, size: PropTypes.number };
 
+export const CogIcon = ({ size, color }) => <FAIcon name="cog" size={size} color={color} />;
+CogIcon.defaultProps = { color: GREY, size: 30 };
```

Comment:
Size of the icons on the menu page were a bit striking to me - I had a [look](https://github.com/oblador/react-native-vector-icons/blob/3458ae8e0d54f3fea821bfc8f5bbb5b67718e60f/lib/icon-button.js#L89) and looks like the originals were "large" - if it was on purpose, I have no preference!

Reply:
https://github.com/openmsupply/mobile/pull/3333/commits/f2515667e7fc0a909ab17f4f460bce8fe354e53e